### PR TITLE
Added modules options to cohort API

### PR DESCRIPTION
**Pull request by tommylau523**

### CODE REVIEW

**Comment by tommylau523** [comment]
* **#1927** <a href="https://app.graphite.dev/github/pr/terraware/terraware-server/1927?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/main/kotlin/com/terraformation/backend/accelerator/model/ModuleDepth.kt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.terraformation.backend.accelerator.model
+
+enum class ModuleDepth {
```
Comment (sgrimm) [inline src/main/kotlin/com/terraformation/backend/accelerator/model/ModuleDepth.kt]: Is this used?

**Review by sgrimm — APPROVED** [review]

File: src/main/kotlin/com/terraformation/backend/accelerator/model/ModuleDepth.kt (hunk 1 above)
Comment (tommylau523) [reply src/main/kotlin/com/terraformation/backend/accelerator/model/ModuleDepth.kt]: Sorry jumbled my PRs a bit. This is supposed to be a later PR to add more fetch options by modules too.

**Comment by tommylau523** [comment]
### Merge activity

* **May 3, 1:32 PM EDT**: @tommylau523 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-server/1927).

**Pull request merged**